### ISSUE TITLE
CX_report.txt appended twice to output file

### ISSUE THREAD

**Issue by martinjvickers** [FelixKrueger/Bismark#150]
I just noticed this after updating to 0.19.0 that the `bismark_methylation_extractor` seems to append `CX_report.txt` twice to the output file.

```
Bismark/bismark_methylation_extractor --comprehensive --cytosine_report --bedGraph --CX test_bismark_bt2.bam --genome_folder TAIR10
>>> Writing genome-wide cytosine report to: test_bismark_bt2.CX_report.txt.CX_report.txt <<<
```

**Comment by FelixKrueger** [comment]
Thanks for spotting.

**Comment by FelixKrueger** [comment]
Addressed here: fadb600a2b85032b3125d293da1957ae94a24e87
